packages/lib/formulas/concatenate/formula.json

```diff
@@ -12,6 +12,17 @@
           { "type": "Array", "ofType": { "type": "Any" } }
         ]
       }
+    },
+     {
+      "name": "1",
+      "formula": { "type": "value", "value": "" },
+      "type": {
+        "type": "Union",
+        "types": [
+          { "type": "String" },
+          { "type": "Array", "ofType": { "type": "Any" } }
+        ]
+      }
```

Comment:
Technically, it could also receive just 1 argument. I don't think we should necessarily change this, but instead use the `variableArguments` flag to somehow indicate in the docs that you can pass in multiple parameters.
```suggestion
```